src/api/net.rs

```diff
@@ -36,6 +36,11 @@ impl<T: Transport> Net<T> {
         CallFuture::new(self.transport.execute("net_peerCount", vec![]))
     }
 
+    /// Returns list of connected nodes parity
+    pub fn parity_peer_list(&self) -> CallFuture<PeerType, T::Out>{
```

Comment:
Let's move this to `parity_` namespace then

Reply:
will do! :) I think it's ``ParitySet``? Didn't realize you gave these instructions. Asked if i had the go ahead on [#198](https://github.com/tomusdrw/rust-web3/pull/198)

Reply:
Sounds good!